### DISCUSSION TITLE
dynamic import is not working when the imported library has been listed in externals.

### DISCUSSION THREAD

**Discussion by songshuzhong** [webpack/webpack#15388] [Q&A]
# Bug report

**What is the current behavior?**
dynamic import is not working when the imported library has been listed in externals.

**If the current behavior is a bug, please provide the steps to reproduce.**

1. import
      import(/*webpackChunkName: "js-export-excel", webpackPrefetch: true */ 'js-export-excel')
        .then(res => {
          const options = {
            name: props.title || 'sheetjs',
            datas: [{
              sheetData,
              sheetName: props.title || 'sheetjs',
              sheetHeader: sheetLabel
            }]
          };
          new res.default(options).saveExcel();
        }).catch(e => {
          proxy.$message.error(e.message);
          console.error(e);
        });
2. config
    module.exports = {
        externals: {
            'js-export-excel': 'js-export-excel'
      }
}；
3.result
Promise.resolve(/*! import() */).then(__webpack_require__.t.bind(null, /*! js-export-excel */ "js-export-excel", 7)).then(function (res) {
        var options = {
          name: props.title || 'sheetjs',
          datas: [{
            sheetData: sheetData,
            sheetName: props.title || 'sheetjs',
            sheetHeader: sheetLabel
          }]
        };
        new res.default(options).saveExcel();
      }).catch(function (e) {
        proxy.$message.error(e.message);
        console.error(e);
      });

**What is the expected behavior?**

**Other relevant information:**
webpack version:  4.44.2
Node.js version: 14.18.1
Operating System: mac
Additional tools:

**Comment by alexander-akait** [comment]
Please create reproducible test repo, thanks
